src/components/config/validate.ts

```diff
@@ -435,3 +445,22 @@ function validateTCPConfig(tcpConfig?: Socket) {
 
   return validationError?.error?.message
 }
+
+function validateRedisConfig(redisConfig?: Redis[]) {
+  if (!redisConfig) {
+    return ''
+  }
+
+  const schema = Joi.object({
+    host: Joi.string().required(),
+    port: Joi.number().required(),
```

Comment:
maybe we can validate the port here, it should be in the range of 0-65536, right? 

```
Joi.number().integer().min(0).max(65536)
```